src/main_options.cpp

```diff
@@ -166,8 +166,16 @@ bool PrintSystemInformation()
     SDL_version sdl_linked_ver;
     SDL_GetVersion(&sdl_linked_ver);
 
-    printf("SDL version (compiled):  %d.%d.%d\n", SDL_MAJOR_VERSION, SDL_MINOR_VERSION, SDL_PATCHLEVEL);
-    printf("SDL version (linked):    %d.%d.%d\n", sdl_linked_ver.major, sdl_linked_ver.minor, sdl_linked_ver.patch);
+    std::cout
+      << "SDL version (compiled):  "
+      << SDL_MAJOR_VERSION << '.' << SDL_MINOR_VERSION << '.' << SDL_PATCHLEVEL
+      << std::endl;
+    std::cout
+      << "SDL version (linked):    "
+      << sdl_linked_ver.major << '.'
+      << sdl_linked_ver.minor << '.'
+      << sdl_linked_ver.patch
+      << std::endl;
```

Comment:
For the record, I have nothing against C, but since we are trying to use C++11 in here, ... :)